backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -972,10 +1005,12 @@ public Set<Observation> convertToAOEObservations(
     } else if (surveyData.getSymptoms() != null
         && surveyData.getSymptoms().containsValue(Boolean.TRUE)) {
       symptomatic = true;
+      List<String> positiveSymptoms = getPositiveSymptoms(surveyData.getSymptoms());
+      observations.addAll(convertToSymptomsObservations(positiveSymptoms));
```

Comment:
This is very clean and makes sense to me from a logic perspective. However, does anyone know why we weren't making the distinction between postive and negative symptoms before in the Fhir conversion?

Or was that distinction already being made, just without the helper function?

Reply:
We were only adding the "Yes" symptomatic observation if there were `true` symptom values (see line [973 above](https://github.com/CDCgov/prime-simplereport/pull/8183/files/5f7623b9de400657a870c3201651aedc8bf1e97b#diff-0f13a6fdf468bcfcc079f96410fbd7204be5fded4c21df796d259163c3ea62c5L973)) 

Since that only checks if any symptom value is `true`, I needed to create an additional method to return all the symptoms that were `true` so it's easier to create an `Observation` for each positive symptom.

Let me know if that makes sense or you see any other optimization! 😸

Reply:
I think this is a tiny bit unclear - I would have initially assumed `Positive` and `Negative` meant that it was a positive or negative test result.

Reply:
I'll rename this and the variable... maybe something like `trueSymptoms` and `filterTrueSymptoms`

Reply:
Ok I updated this! Let me know if that makes it more clear! 😸

Reply:
Small nit for the name, I think `symptomsPresent` and `getFilteredSymptomsPresent` would be clearer than "true" and be consistent with the clinical context of patients presenting with certain symptoms

Reply:
Thank you for the suggestion! I like those better! Naming is hard 😅